prog.go

```diff
@@ -613,6 +613,26 @@ func (p *Program) ID() (ProgramID, error) {
 	return ProgramID(info.id), nil
 }
 
+// ProgramStats holds statistics of a program.
+//
+// Values can be 0 if collecting of statistics is not enabled.
+type ProgramStats struct {
```

Comment:
Since this is an exported struct, IMO these fields should be properly capitalized and expanded as is the convention in Go: `RuntimeNS` and `RunCount`.